bin/start-history-server.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+set -e
+
+export THS_HOME="$(cd "`dirname "$0"`"/..; pwd)" # root of the project
```

Comment:
I think we may want to reuse some of these variables/settings client-side, too. What do you think about renaming these variables/settings `TONY_HOME`, `TONY_CONF_DIR`, `TONY_CLASSPATH`, `tony-site.xml`, etc.?

Reply:
https://github.com/linkedin/TonY/pull/29#discussion_r228000420